### PR TITLE
Add InfluxDB v2 links

### PR DESCRIPTION
**Pull request by sanderson**
I updated the homepage and product drop-down to include links to the InfluxDB v2 documentation.

**New homepage**
image

**New product dropdown**  
image

### CODE REVIEW

File: data/home_cards.yaml [hunk 1]
```diff
@@ -1,9 +1,26 @@
 cards:
 
+  # InfluxDB v2 card
+  - id: influxdb2
+    category: oss
+    title: InfluxDB v2.0
```
Comment (kelseiv) [inline data/home_cards.yaml]: InfluxDB 2.0

File: data/home_cards.yaml [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+    description: InfluxDB v2.0 provides everything you need from a time series platform in a single binary.
+    links:
+      - text: Download InfluxDB v2.0
```
Comment (kelseiv) [inline data/home_cards.yaml]: InfluxDB 2.0 (or v2.0.0-alpha)

**Comment by kelseiv** [comment]
Scott, couple style issue to verify:
- see alpha text (no button) in the dropdown:
Screen Shot 2019-10-07 at 5 16 39 PM
- don't see purple background on influx v2 card. See this for dark view:
Screen Shot 2019-10-07 at 5 18 25 PM
 changes slightly for light view:
Screen Shot 2019-10-07 at 5 18 54 PM
- no matter how wide I expand screen, Flux stays on second line
PS am using Chrome

**Comment by timhallinflux** [comment]
I'd like to see Platform|v2.0|Flux "locked" on that top line. 

The rest looks fine.

**Comment by sanderson** [comment]
@kelseiv You need to hard refresh (command + shift + r) to clear the page cache and load the new stylesheets.

@timhallinflux They are locked to the top row until the screen width narrows and the responsive styles kick in.

File: data/home_cards.yaml (hunk 2 above)
Comment (kelseiv) [reply data/home_cards.yaml]: Makes sense. I'd drop v then.

**Pull request merged**